esmvalcore/_citation.py

```diff
@@ -0,0 +1,208 @@
+"""Citation module."""
+import os
+import logging
+import re
+from pathlib import Path
+import requests
+
+from ._config import DIAGNOSTICS_PATH
+
+if DIAGNOSTICS_PATH:
+    REFERENCES_PATH = Path(DIAGNOSTICS_PATH) / 'references'
+else:
+    REFERENCES_PATH = ''
+
+logger = logging.getLogger(__name__)
+
+CMIP6_URL_STEM = 'https://cera-www.dkrz.de/WDCC/ui/cerasearch'
+
+# it is the technical overview and should always be cited
+ESMVALTOOL_PAPER_TAG = 'righi19gmdd'
+ESMVALTOOL_PAPER = (
+    '@article{righi19gmdd,\n\t'
```

Comment:
@bouweandela thanks, a tab is needed for contents in the bibtex format.

Reply:
Ok, I understand. Could you move the tabs to the beginning of the line that they are on? I think this will be easier to read if you always put newline characters at the end of a line.